latch_cli/services/move.py

```diff
@@ -0,0 +1,110 @@
+import click
+import gql
+from gql.transport.exceptions import TransportQueryError
+
+from latch.gql._execute import execute
+from latch_cli.services.cp.ldata_utils import LDataNodeType, get_node_data
+from latch_cli.services.cp.path_utils import (
+    get_name_from_path,
+    get_path_error,
+    is_remote_path,
+)
+
+
+def move(
+    src: str,
+    dest: str,
+):
+    if not (is_remote_path(src) and is_remote_path(dest)):
```

Comment:
```suggestion
    if not is_remote_path(src) or not is_remote_path(dest):
```
idk kinda reads better to me